pkg/specgen/specgen.go

```diff
@@ -551,7 +551,7 @@ func NewSpecGenerator(arg string, rootfs bool) *SpecGenerator {
 	csc := ContainerStorageConfig{}
 	if rootfs {
 		csc.Rootfs = arg
-		// check if rootfs is actually overlayed
+		// check if rootfs is should use overlay
```

Comment:
I think this should be corrected to `is actually overlay` (remove the `ed`), but if not, please remove the `is`